posthog/models/filters/mixins/funnel.py

```diff
@@ -172,11 +172,25 @@ def funnel_step_to_dict(self):
 
 class FunnelPersonsStepBreakdownMixin(BaseParamMixin):
     @cached_property
-    def funnel_step_breakdown(self) -> Optional[Union[str, int]]:
+    def funnel_step_breakdown(self) -> Optional[Union[List[str], int, str]]:
         """
         The breakdown value for which to get persons for.
+
+        For person and event properties as this value is set within the funnel it is always an array.
+        Until multi property breakdowns is released it is always a single value array
+
+        for groups it is always a string
+
+        for cohorts it is always an int
         """
-        return self._data.get(FUNNEL_STEP_BREAKDOWN)
+        raw: Optional[str] = self._data.get(FUNNEL_STEP_BREAKDOWN)
+        if not raw:
+            return raw
+
+        try:
+            return json.loads(raw)
+        except (TypeError, json.decoder.JSONDecodeError):
+            return raw
```

Comment:
non-blocking

Is it worth always returning an array here? Not sure of the ramifications downstream though, fine to get in as is.

Reply:
I'm 90% sure I'm going to end up looping back once I've looked at Trends!